src/main/java/seedu/address/logic/commands/DeleteMemberCommand.java

```diff
@@ -72,6 +79,22 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         Group modifiedGroup = new Group(toModify.getName(), toModify.asUnmodifiableObservableList());
+
+        List<CourseMate> courseMateList = new ArrayList<>();
+        int index = 0;
+        for (List<CourseMate> courseMateDeleteList: courseMateSet) {
```

Comment:
There is significant code duplication here. Might be a sign that you should create new classes for this.